pages/who-we-are/corporate/Job/information-pack.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: basic
+title: Job Information Pack
+permalink: /who-we-are/corporate/jobs/job-information-pack
```

Comment:
Please rename the new folder to 'jobs' to match the url, and move jobs.md into the new jobs folder